### PR TITLE
Fix runner for monado

### PR DESCRIPTION
**Pull request by Hyjale**
- The runner was not building a shared object plugin for the runtime due to a missed line. 
- I have added OpenVINS to SLAM config file to allow for more accessibility (digging through to find the correct version would be a pain for most people).

### CODE REVIEW

**Comment by charmoniumQ** [comment]
Good find.

**Review by charmoniumQ — APPROVED** [review]

File: configs/rt_slam_plugins.yaml [hunk 1]
```diff
@@ -5,5 +5,9 @@ plugin_group:
     path:
       git_repo: https://github.com/ILLIXR/Kimera-VIO
       version: 6ea99d33fc2435a8c514049416dcb2f8a0e31692
+  #- name: OpenVINS
+  #  path:
+  #    git_repo: https://github.com/ILLIXR/open_vins.git
+  #    version: c6a4f8835851d21f87add38260e73df63c6354a1
```
Comment (apache8080) [inline configs/rt_slam_plugins.yaml:11]: @Hyjale can you remove this. We decided to not have openvins in the config file. A user can add this themselves.

@mhuzai @charmoniumQ unless we want to change this.

Comment (apache8080) [reply configs/rt_slam_plugins.yaml:11]: We had a separate file for OV before.

Comment (apache8080) [reply configs/rt_slam_plugins.yaml:11]: We can just let this through now and change it later.

**Review by apache8080 — APPROVED** [review]

**Pull request merged**